tbb.sh

```diff
@@ -0,0 +1,35 @@
+package: TBB
+version: "v2021.5.0"
+tag: v2021.5.0
+source: https://github.com/oneapi-src/oneTBB/
+build_requires:
+ - "GCC-Toolchain:(?!osx)"
+ - CMake
+prefer_system: .*
+prefer_system_check: |
+  printf "#include <tbb/concurrent_unordered_map.h>\n static_assert(TBB_INTERFACE_VERSION >= 11009, \"min version check failed\");\n" | c++ -std=c++11 -xc++ - -c -o /dev/null
+---
+
+#!/bin/bash -e
+cmake $SOURCEDIR -DCMAKE_INSTALL_PREFIX=$INSTALLROOT   \
+          ${CXXSTD:+-DCMAKE_CXX_STANDARD=$CXXSTD} -DTBB_TEST=OFF
+
+# Build and install
+cmake --build . -- ${JOBS:+-j$JOBS} install
+
+# make soft link of lib64 dir to lib
+pushd "$INSTALLROOT"
+  # Fix ambiguity between lib/lib64
+  if [[ ! -d lib && -d lib64 ]]; then
+    ln -nfs lib64 lib
+  elif [[ -d lib && ! -d lib64 ]]; then
+    ln -nfs lib lib64
+  fi
```

Comment:
Why not simply using `-DCMAKE_INSTALL_LIBDIR=lib`?

Reply:
no idea. I took the inspiration from different recipes in alidist

Reply:
Thank you for your update. Indeed we should probably fix also the other recipes.